lib/suites/LinkedDataSignature.js

```diff
@@ -50,6 +70,53 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
     this.useNativeCanonize = useNativeCanonize;
   }
 
+  /**
+   * See constructor docstring for param details.
+   *
+   * @returns {{verificationMethod: string, key: LDKeyPair, signer: Function,
+   *   verifier: Function}} - Validated and initialized key-related parameters.
+   */
+  _validateVerificationMethod({verificationMethod, key, signer, verifier}) {
+    if(!(key || signer || verifier)) {
+      // eslint-disable-next-line max-len
+      throw new TypeError('Either a "key" or at least one of "signer"/"verifier" parameters is required');
```

Comment:
```suggestion
      throw new TypeError('Either "key" or one of "signer"/"verifier" is required.');
```

Reply:
@dlongley - I think the 'at least' part is important, since you can pass in both. No?